MVP/Resources/Translations.de.resx

```diff
@@ -0,0 +1,456 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>1.3</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="about_licenses_description" xml:space="preserve">
+    <value>Diese App steht auf den Schultern von Riesen. Ohne sie wäre die Entstehung dieser App nicht möglich gewesen. Genauere Infos zu diesen aussergewöhnlichen Persönlichkeiten finden sie bei den Lizenzen der grossartigen Bibliotheken, welche in dieser App verwendet werden.</value>
+  </data>
+  <data name="about_licenses_title" xml:space="preserve">
+    <value>Lizenzen</value>
+  </data>
+  <data name="about_opensource_description" xml:space="preserve">
+    <value>Diese App ist komplett Open Source. Wenn die App auf gefallen stösst, kannst Du gerne ein GitHub-Sponsor werden und so mir eine digitale Tasse Kaffee pro Monat kaufen. ☕️</value>
+  </data>
+  <data name="about_opensource_title" xml:space="preserve">
+    <value>Open Source</value>
+  </data>
+  <data name="about_title" xml:space="preserve">
+    <value>Über diese App</value>
+  </data>
+  <data name="about_why_description" xml:space="preserve">
+    <value>Es gab schon immer ein Bedürfnis für Microsoft MVPs, ihre Community-Aktivitäten auf einfachere Weise zu verwalten. Das Online-Portal funktioniert gut, aber ein ähnliches Ding in der Tasche zu haben, um Aktivitäten on the fly zu erstellen, ist wirklich cool. Und genau dafür ist dieses Projekt da!</value>
```

Comment:
aber etwas vergleichbares in der Tasche zu haben, um Aktivitäten on the fly zu erfassen,